docs/getting-started/3-rulesets.md

```diff
@@ -1,161 +1,60 @@
+## Extend an Existing Ruleset
 
-### Rules
+The fastest way to create a ruleset is to use the `extends` property to leverage an existing ruleset.
 
-Rules might look a bit like this:
+Spectral comes with two built-in rulesets:
 
-```yaml
-rules:
-  paths-kebab-case:
-    description: Paths should be kebab-case.
-    message: "{{property}} should be kebab-case (lower-case and separated with hyphens)"
-    severity: warn
-    given: $.paths[*]~
-    then:
-      function: pattern
-      functionOptions:
-        match: "^(\/|[a-z0-9-.]+|{[a-zA-Z0-9_]+})+$"
-```
-
-Spectral has [built-in functions](../reference/functions.md) such as `truthy` or `pattern`, which can be used to power rules.
-
-Rules then target certain chunks of the JSON/YAML with the `given` keyword, which is a [JSONPath](http://jsonpath.com/) (actually, we use [JSONPath Plus](https://www.npmjs.com/package/jsonpath-plus)).
-
-The example above adds a single rule that looks at the root level `tags` object's children to make sure they all have a `description` property.
-
-### JSONPath Plus
-
-As mentioned, spectral is using JSONPath Plus which expands on the original JSONPath specification to add some additional operators and makes explicit some behaviors the original did not spell out.
-
-Here are some convenient **additions or elaborations**:
-
-- `^` for grabbing the **parent** of a matching item
-- `~` for grabbing **property names** of matching items (as array)
-- **Type selectors** for obtaining:
-  - Basic JSON types: `@null()`, `@boolean()`, `@number()`, `@string()`, `@array()`, `@object()`
-  - `@integer()`
-  - The compound type `@scalar()` (which also accepts `undefined` and
-    non-finite numbers when querying JavaScript objects as well as all of the basic non-object/non-function types)
-  - `@other()` usable in conjunction with a user-defined `otherTypeCallback`
-  - Non-JSON types that can nevertheless be used when querying
-    non-JSON JavaScript objects (`@undefined()`, `@function()`, `@nonFinite()`)
-- `@path`/`@parent`/`@property`/`@parentProperty`/`@root` **shorthand selectors** within filters
-- **Escaping**
-  - `` ` `` for escaping remaining sequence
-  - `@['...']`/`?@['...']` syntax for escaping special characters within
-    property names in filters
-- Documents `$..` (**getting all parent components**)
-
-### Extending Rulesets
-
-Rulesets can extend other rulesets using the `extends` property, allowing you to pull in other rulesets.
-
-```yaml
-extends: spectral:oas
-```
+- `spectral:oas` - [OpenAPI v2/v3 rules](./4-openapi.md)
+- `spectral:asyncapi` - [AsyncAPI v2 rules](./5-asyncapi.md)
 
-Extends can reference any [distributed ruleset](../guides/7-sharing-rulesets.md). It can be a single string, or an array of strings, and can contain either local file paths, URLs, or even npm modules.
+To create a ruleset that extends both rulesets, in your terminal run:
```

Comment:
Punctuation is a little off here. I think you need a comma between "in your terminal" and "run"